include/extractor/extraction_way.hpp

```diff
@@ -50,6 +65,20 @@ struct ExtractionWay
     void set_backward_mode(const TravelMode m) { backward_travel_mode = m; }
     TravelMode get_backward_mode() const { return backward_travel_mode; }
 
+    // wrappers to allow assigning nil (nullptr) to string values
+    void SetName(const char* value) { detail::maybeSetString(name, value); }
```

Comment:
is there a way we could also pass the destination to write into from, so we don't have to replicate this behaviour for all string types?